kobuki_node/package.xml

```diff
@@ -1,8 +1,8 @@
-<package>
+<package format="2">
```

Comment:
Should we get the xml schema lines in as well?

Reply:
Added in 9605787.